doc/extending/xlsx/xlsx_import.md

```diff
@@ -60,8 +60,11 @@ NW = no warning generated for these conversions.
   - most border styles. We only support thin borders, and will convert every other border style to thin border.
   - diagonal borders
 - Align :
-  - some horizontal alignments. We only support left/right/center.
-  - vertical alignement
+  - some horizontal alignments. We only support left/right/center. Other types will be converted as follows:
+    - general -> default (currently left)
```

Comment:
```suggestion
```
Our `default` horizontal alignment have the same behaviour as excel `general`. Texts are aligned left, numbers are aligned right.